### PR TITLE
Initial Faces 5.0 FAT

### LINKED ISSUE #25783: Faces 5.0: Create new Faces 5.0 FAT

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #25783

### CODE REVIEW

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

File: dev/io.openliberty.org.apache.myfaces.5.0_fat/fat/src/io/openliberty/org/apache/myfaces50/fat/JSFUtils.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
```
Comment (volosied) [inline dev/io.openliberty.org.apache.myfaces.5.0_fat/fat/src/io/openliberty/org/apache/myfaces50/fat/JSFUtils.java:11]: I can remove this when I add tests.

**Review by volosied — APPROVED** [review]

**Comment by volosied** [comment]
#libby

**Pull request merged**